test/d2l-scroll-wrapper.html

```diff
@@ -4,7 +4,6 @@
 		<meta charset="utf-8">
 		<title>wrapper tests</title>
 		<meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
-		<script src="../../@babel/polyfill/browser.js"></script>
```

Comment:
Why was this removed?

Reply:
This was something that was removed from the conversion script a while ago, though I am not sure of the reason.  @dlockhart : do you know the background on that?

Reply:
I think it was causing issues and was determined to no longer be necessary:
https://github.com/Brightspace/polymer-3-converter/pull/91